translations/en.json

```diff
@@ -25,8 +25,6 @@
     "add": "Add",
     "added": "added",
     "add-a-watch-account": "Add a watch-only address",
-    "add-account": "Add an account",
-    "add-account-description": "You can import any type of Ethereum account to add it to your Status wallet",
```

Comment:
These translations are used in old wallet screens. Removing these translations would break the old screens.
(We agreed to keep the old wallet design until the new wallet design is completely done.)

Reply:
yes, I think we should mark the keys as old instead? 🤔 or -OLD to be even more obvious. something standard like this? 
on a sidenote @briansztamfater let's create that issue to clean up the translation strings file like we discussed a while back! :)

Reply:
@smohamedjavid "add-account-description" is not used anywhere. 
The new and old text of "add-account" is almost the same, so not worth changing it I think

Reply:
Thanks for checking @OmarBasem